internal/validations/validations_os.go

```diff
@@ -0,0 +1,43 @@
+package validations
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/openshift/check-payload/internal/types"
+)
+
+func ValidateOperatingSystem(mountPath string) types.OperatingSystemInfo {
+	info := types.OperatingSystemInfo{
+		Certified: false,
+		Error:     nil,
+	}
+
+	path := filepath.Join(mountPath, "/etc/redhat-release")
+	info.Path = path
+	f, err := os.Open(path)
+	if err != nil {
+		e := errors.New("release file not found")
+		info.Error = e
```

Comment:
nit: why do you need an intermediate variable (`e`) here?

Reply:
Also, it makes sense to return the actual error here (which can help to figure out what is going on).